apps/spruce/README.md

```diff
@@ -177,66 +180,46 @@ way to do so is to populate the local db using real data from the staging or
 production environments.
 
 1. You should identify if the data you need is located in the staging or prod db
-   and ssh into them (You should be connected to the office network or vpn
-   before proceeding). The urls for these db servers can be located in the
-   `fabfile.py` located in the evergreen directory or
-   [here](https://github.com/10gen/kernel-tools/blob/master/evergreen/fabfile.py).
-2. You should ensure you are connected to a secondary node before proceeding.
-3. Run `mongo` to open the the mongo shell.
-4. Identify the query you need to fetch the data you are looking for.
+   and connect to them using the instructions in the Evergreen Operations Guide
+2. Run `mongo` to open the the mongo shell.
+3. Identify the query you need to fetch the data you are looking for.
 
    ```
-   mci:SECONDARY> rs.secondaryOk() // Allows read operations on a secondary node
-   mci:SECONDARY> use mci // use the correct db
-   switched to db mci
-   mci:SECONDARY>  db.distro.find({_id: "archlinux-small"}) // the full query
+   Atlas atlas-mxabkq-shard-0 [primary] mci> db.distro.find({_id: "archlinux-small"}) // the full query
    ```
 
-5. Exit from the mongo shell and prepare to run `mongoexport`
+4. Write the file to your local system using the following command
 
    ```
-   mongoexport --db=mci --collection=distro --out=distro.json --query='{_id: "archlinux-small"}'
-   2020-07-29T17:41:50.266+0000	connected to: localhost
-   2020-07-29T17:41:50.269+0000	exported 1 record
+   fs.writeFileSync('output.json', JSON.stringify(db.distro.find({_id: "archlinux-small"})))
```

Comment:
This is from within the mongo shell right? If so I think including the same prompt text as above would clarify that.